### DISCUSSION TITLE
Something Like PayPal Authorized Payment And Voidable Later

### DISCUSSION THREAD

**Discussion by martiankoder** [btcpayserver/btcpayserver#5089] [Ideas / Feature Requests]
Hi,
 It's would be great if BTCPay supports something like PayPal's Authorized Payment and can be voided/cancelled later to avoid refunding fees. This feature is so critical for an online shop. A shop owner can check inventory after a payment is made, if the item cannot be fulfilled, then the Authorized Transaction can be voided and customer gets back the fund without any fees. Do you know how to set this up in BTCPay server, or documents on coding this ? Thank you.

**Comment by Kukks** [comment]
You've already posted this on a number of issues. Please stop spamming the
feature request everywhere.
